extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -208,8 +210,12 @@ public void configurationDescriptorBuilding(
         for (PersistenceXmlDescriptorBuildItem persistenceXmlDescriptorBuildItem : persistenceXmlDescriptors) {
             allDescriptors.add(persistenceXmlDescriptorBuildItem.getDescriptor());
         }
-        handleHibernateORMWithNoPersistenceXml(allDescriptors, resourceProducer, systemPropertyProducer,
-                defaultJdbcDataSourceBuildItem, applicationArchivesBuildItem, launchMode.getLaunchMode());
+
+        final boolean hibernateReactivePresent = capabilities.isCapabilityPresent(Capabilities.HIBERNATE_REACTIVE);
+        if (!hibernateReactivePresent) {
```

Comment:
This way the mere presence of the `HIBERNATE_REACTIVE` capability would break existing ORM applications, I don't think we can have that. Can you think of a better way?

Reply:
For now, simultaneous usage of Hibernate ORM and Reactive is not really tested, and therefore not supported (yet), so that is a current limitation.
Existing ORM apps should still work if they have persistence.xml based config, but once they pull in HR it is assumed that the default PU is a reactive one.
Perhaps we could key this off of the default datasource configuration also? So if the default JDBC datasource is NOT enabled (i.e. `quarkus.datasource.jdbc=false`) then this can stay as-is, but if the default JDBC datasource IS enabled, then we'd still want to produce this somehow. But this goes back to the current limitation of not being able to use ORM and HR at the same time.

Reply:
ok, fair enough - especially as I don't have a better idea to suggest :)